web-common/src/features/canvas/components/charts/BaseChart.ts

```diff
@@ -108,13 +112,27 @@ export abstract class BaseChart<
     );
 
     const timeGrain = get(this.timeAndFilterStore)?.timeGrain;
+    const showTDD = canLinkTimeDimensionDetail(spec, this.type);
 
     return {
       whereFilter: dimensionFilters,
       dimensionThresholdFilters,
       showTimeComparison: false,
-      activePage: DashboardState_ActivePage.PIVOT,
+      activePage: showTDD.canLink
```

Comment:
The variables names feels a little weird. `tddLink.canLink` might read better than `showTDD.canLink`

`canLinkTimeDimensionDetail` should also be renamed to match.